aim/web/ui/src/pages/RunDetail/ImagesVisualizer/ImagesVisualizer.tsx

```diff
@@ -135,6 +135,7 @@ function ImagesVisualizer(
             additionalProperties={additionalProperties}
             onActivePointChange={onActivePointChange}
             illustrationConfig={{ title: 'No Tracked Images' }}
+            selectOptions={[]}
```

Comment:
maybe we can set `selectOptions` optional and set an empty array in the component when destructuring the props